.storybook/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
https://github.com/storybookjs/storybook/issues/15335
Some of the storybook packages don't have an `exports` field, so webpack ends up complaining.
This `package.json` is there so webpack ignores `"type": "module"` in the real `package.json`. 🤷‍♂️